semperfiwebdesign/all-in-one-seo-pack
# Add Schema structural data

Issue #73

## Proposed changes

Add the schema.org structural data to be outputted on the frontpage.

## Types of changes

What types of changes does your code introduce?
_Delete those that don't apply_

- New feature (non-breaking change which adds functionality)
- Adds new API hooks

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code. If creating a separate issue for an item, link to the issue # at the end of the line._


## Testing instructions

1. Using defaults, navigate to one of the front pages.
2. Right click and inspect to show code.
3. In the header, underneath ``, there should be a script for ld+json.
4. Copy the data, and go to [Google Structured Data Testing Tool](https://search.google.com/structured-data/testing-tool).
5. Paste code and click on Validate.

To change sitewide settings.

1. Go to admin, and click on AIOSEOP > General Settings.
2. Navigate to Schema Settings, and change settings there.

## Further comments

This will still need help details added to tooltips with help from @wpsmort 

Note: I have noticed the phone number for contactPoint does require the full phone number including a country's code ( ex. +1 for US ), and given that countries use different formats, I wasn't sure if it would be wise to format it ourselves.
